docs/api.js

```diff
@@ -2,21 +2,23 @@ export default {
   swagger: '2.0',
   info: {
     version: '1.0',
-    title: 'NewDev Api Documentation',
-    description: 'NewDev Api Documentation',
+    title: 'NewDev API Docs',
+    description: 'A documentation for our NewDev API.',
```

Comment:
Could we change the description to `NewDev Api Documentation`?

Reply:
Sure!